src/vehicle_use.cpp

```diff
@@ -797,6 +797,32 @@ void vehicle::use_controls( const tripoint &pos )
     }
 }
 
+static void handbrake()
```

Comment:
You should make `handbrake` an exported function instead of copying it around.

Reply:
Thanks. Like this?